domino/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.1.1"
+__version__ = "1.1.2"
```

Comment:
Let's not do the version change yet. We already had 1.1.1 release (in github release and PyPI) for Domino 5.3.0. We can have 1.1.2 for 5.4.0.

Reply:
Good to know, thanks for the tip. I've undo the version update and moved the changelog notes to the "Unreleased" section.

Reply:
@mildmelon version upgrades are done only at release time.